client/pages/[...all].vue

```diff
@@ -1,3 +1,18 @@
+<script setup>
+const { isAuthenticated } = useAuth()
+
+watch(isAuthenticated, (val) => {
+  setPageLayout(val ? 'sidebar' : 'centercard')
+  usePageName(() => '404')
+}, { immediate: true })
```

Comment:
```suggestion
const pageName = usePageName()
watch(isAuthenticated, (val) => {
  setPageLayout(val ? 'sidebar' : 'centercard')
  pageName.value = '404'
}, { immediate: true })
```

When possible, avoid using composables outside of composables or the setup function, to avoid ref leakage